nestkernel/synaptic_element.cpp

```diff
@@ -120,7 +120,10 @@ nest::SynapticElement::set( const DictionaryDatum& d )
   double new_tau_vacant = tau_vacant_;
 
   // Store values
-  updateValue< double >( d, names::growth_rate, growth_rate_ );
+  if ( d->known( names::growth_rate ) )
```

Comment:
So, if the growth rate is in the dictionary, set the value, but if not, propagate the current value up. Is this right, that a set operation can instead change the dictionary passed? Maybe the set should do nothing if not known, and a get is needed to access the value?